src/main/java/wooteco/subway/domain/Station.java

```diff
@@ -9,15 +11,37 @@ public Station() {
     }
 
     public Station(Long id, String name) {
+        validateNull(name);
         this.id = id;
         this.name = name;
     }
 
+    private void validateNull(String name) {
+        if (name.isBlank()) {
+            throw new ClientException("지하철 역의 이름을 입력해주세요.");
+        }
+    }
+
     public Long getId() {
         return id;
     }
 
     public String getName() {
         return name;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        Station station = (Station) o;
+
+        return name != null ? name.equals(station.name) : station.name == null;
```

Comment:
```suggestion
        return Objects.equals(name, station.name);
```

인텔리제이가 이렇게 추천을 해주고 있습니다~!